pkgs/applications/networking/cluster/fluxcd/default.nix

```diff
@@ -1,25 +1,26 @@
 { lib, buildGoModule, fetchFromGitHub, fetchzip, installShellFiles }:
 
 let
-  version = "0.17.2";
+  pkgVersion = "0.17.2";
```

Comment:
```suggestion
  version = "0.17.2";
```
Renaming the others should be fine I guess.

Reply:
Done